omod-common/src/main/java/org/openmrs/module/webservices/rest/web/ConversionUtil.java

```diff
@@ -252,7 +252,7 @@ public static Object convert(Object object, Type toType) throws ConversionExcept
 			} else if (toClass.isAssignableFrom(Locale.class)) {
 				return LocaleUtility.fromSpecification(object.toString());
 			} else if (toClass.isEnum()) {
-				return Enum.valueOf((Class<? extends Enum>) toClass, object.toString());
+				return Enum.valueOf((Class<? extends Enum>) toClass, object.toString().toUpperCase());
```

Comment:
@rkorytkowski @dkayiwa what do you think about it? I think it is more convenient to convert enums case-insensitively